config-scripts/cups-compiler.m4

```diff
@@ -98,7 +98,7 @@ if test -n "$GCC"; then
 	if test -z "$OPTIM"; then
 		if test "x$with_optim" = x; then
 			# Default to optimize-for-size and debug
-       			OPTIM="-Os -g"
+       			OPTIM="-g"
```

Comment:
The default compiler options need to specify optimization. Specify --with-optim="-g" to force a debug-only build, or to pass any other optimization options.

Reply:
Ok, maybe, I would understand if you specify -O2 by default, ok, it is understandable, it is also by convention, like a trade-off in optimization levels. But -Os looks weird. No offence given

Reply:
take a look how [it is done in cmake](https://cmake.org/cmake/help/v3.0/variable/CMAKE_BUILD_TYPE.html)
it can be food for mind